tests/test_misc.py

```diff
@@ -71,9 +71,10 @@ def test_file_load_json(mocker, testdatadir) -> None:
     ("ETHH20", 'ETHH20'),
     (".XBTBON2H", '_XBTBON2H'),
     ("ETHUSD.d", 'ETHUSD_d'),
-    ("ADA-0327", 'ADA_0327'),
-    ("BTC-USD-200110", 'BTC_USD_200110'),
-    ("F-AKRO/USDT", 'F_AKRO_USDT'),
+    ("ADA-0327", 'ADA-0327'),
+    ("BTC-USD-200110", 'BTC-USD-200110'),
+    ("BTC-PERP:USDT", 'BTC-PERP_USDT'),
```

Comment:
Don't worry about symbols like this, FTX will also be getting the unified symbols

Reply:
Today? tomorrow? or sometime next month?

Sorry, but I am looking at current state - not at something that may happen sometime in the future.

Reply:
binance and ftx are the only different ones, I see a 0% chance of implementing everything thats needed for FTX on freqtrade before that happens

Reply:
as said - it's there, no more code will be needed to support this - so i don't see a reason to remove it at this point in time.

Reply:
pretty quickly after the binance changes get merged, the functionality is almost identical, I'm just saying don't waste your time on the PERP symbols